tests/test_sync.py

```diff
@@ -0,0 +1,73 @@
+import json
+from pytest import fixture
+import os
+
+from patientsearch.models import sync_bundle
+
+
+@fixture
+def external_patient_search(request):
+    data_dir, _ = os.path.splitext(request.module.__file__)
```

Comment:
might be some way to configure/read this dir from pytest. I found a plugin, but maybe there's some core pytest config

>pytest-datadir will look up for a directory with the name of your module or the global 'data' folder

Reply:
nice find!  plugged in

Reply:
Nice! I couldn't find any config to it in pytest, does it just work transparently?

Reply:
once installed - pytest does lots of magic includes, mocker is another example.